recipes/configs/qwen2_5/32B_lora.yaml

```diff
@@ -0,0 +1,125 @@
+# Config for multi-device LoRA finetuning in lora_finetune_distributed.py
```

Comment:
Why do we only have 32B lora distributed?

Reply:
I figured at that model size people would almost always be doing a distributed LoRA, right? I got OOMs for the other configs